packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -105,6 +117,23 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// {@macro flutter.widgets.RawAutocomplete.initialValue}
   final TextEditingValue? initialValue;
 
+  /// If the options view overlay should be shown when the options returned from
+  /// [optionsBuilder] is empty.
+  ///
+  /// Set to true to display a custom options view on
+  /// first focus or to display a message if no options are found.
+  ///
+  /// If not provided, will only display an options view when
+  /// some options have been returned from [optionsViewBuilder].
+  final bool showOptionsViewOnEmptyOptions;
+
+  /// If the options view overlay should be shown while the [optionsBuilder]
+  /// future waits to be resolved.
+  ///
+  /// Set to true to display a loading message in the options view while waiting
+  /// for options to be fetched over the network.
```

Comment:
I would add a "See also" section in each of these two new parameters that points to the other parameter.